study-builder/studies/mbc/blood-consent.conf

```diff
@@ -67,7 +67,15 @@
     user.studies["cmi-mbc"].forms["BLOODCONSENT"].questions["BLOODCONSENT_FULLNAME"].answers.hasText()
     && user.studies["cmi-mbc"].forms["BLOODCONSENT"].questions["BLOODCONSENT_DOB"].answers.hasDate()
   """,
-  "elections": [],
+  "elections": [
+    {
+      "stableId": "BLOODCONSENT",
+      "selectedExpr": """
```

Comment:
We don't have any question with stableID to track consented or not since this is Blood consent activity 
Having election this way seem to work fine.

Reply:
I recall Ropes said if they completed blood/tissue consent then they have given consent for that. Maybe we can do
```
user.studies["cmi-mbc"].forms["BLOODCONSENT"].isStatus("COMPLETE")
```
for the `selectedExpr` (likewise for tissue)?